app/globals.css

```
@@ -1,26 +1,30 @@
 @import "tailwindcss";
 
 :root {
-  --background: #f7f8fc;
-  --surface: #eef1f7;
-  --foreground: #172033;
+  --radius: 1rem;
+  --font-body:
+    "Pretendard Variable", Pretendard, -apple-system, BlinkMacSystemFont,
```

Stylelint (17.14.0)

[error] Expected "Pretendard" to be "pretendard" (value-keyword-case)

(value-keyword-case)